### PR TITLE
Complete test blog

### PR DESCRIPTION
**Pull request by shhetri**

### CODE REVIEW

File: _posts/2015-08-05-test-blog.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+layout: post
+title: "Test Blog Post"
+author: sumit_chhetri
+modified:
+excerpt: "My first blog post"
+tags: []
+---
+
+### This is a test blog post
+I am testing the blog post.
```
Comment (geshan) [inline _posts/2015-08-05-test-blog.md:11]: at least show some markdown :) like
- one
- two

File: _posts/2015-08-05-test-blog.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: "Test Blog Post"
+author: sumit_chhetri
+modified:
```
Comment (geshan) [inline _posts/2015-08-05-test-blog.md:5]: date missing

File: _data/authors.yml [hunk 3]
```diff
@@ -25,3 +25,10 @@ sweta_shrestha:
   bio: "UX Developer"
   avatar: bio-photo.jpg
   twitter: sweet_anjell
+
+sumit_chhetri:
+  name: Sumit Chhetri
+  email: er.sumit.chhetri@gmail.com
```
Comment (geshan) [inline _data/authors.yml:31]: use YIPL email :)

File: _posts/2015-08-05-test-blog.md (hunk 1 above)
Comment (shhetri) [inline _posts/2015-08-05-test-blog.md:11]: I will dai in the actual demo

This pull request can be deleted.

**Comment by geshan** [comment]
can be closed :)

**Comment by shhetri** [comment]
yeah! :)

**Pull request closed without merging**